Makefile

```diff
@@ -111,10 +111,13 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
+	$(KUSTOMIZE) build config/operand_rbac | kubectl apply -f -
 
 undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build config/operand_rbac | kubectl delete -f -
 	$(KUSTOMIZE) build config/default | kubectl delete -f -
```

Comment:
Does this work properly?  If I understand correctly, the first `kubectl delete` command would delete the namespace because the namespace definition is in `config/operand_rbac/operand_rbac.yaml`, but deleting the namespace would also delete the objects inside the namespace, so the objects that the second `kubectl delete` command is intended to delete would already deleted by the first `kubectl delete` command.  

Maybe just deleting the namespace would be sufficient for now.  On the other hand, we might want the operator to have the chance to do some cleanup before RBAC and the operator itself are deleted.  If that isn't currently a requirement though, we can just do the simplest thing (deleting the namespace) for now if that suffices.  

Should this rule or another rule clean up the operands that the operator has created?

Reply:
Hi @Miciah , I put the operand namespace (+contents) delete first so that I can ensure that when `make undeploy` is run, all roles and namespace are also gone. Strategy being to remove operand first, operator later.
If we want to let the operator cleanup first, then remove operand namespace, I can simply reverse the order.